feature/repair/src/main/java/com/iteneum/repair/presentation/RepairView.kt

```diff
@@ -8,34 +8,58 @@ import androidx.compose.ui.Modifier
 import androidx.compose.ui.res.stringArrayResource
 import androidx.compose.ui.res.stringResource
 import androidx.core.text.isDigitsOnly
+import androidx.hilt.navigation.compose.hiltViewModel
+import com.iteneum.RepairModel
 import com.iteneum.designsystem.components.*
 import com.iteneum.designsystem.theme.LPTypography
 import com.iteneum.designsystem.theme.LeasePertTheme
 import com.iteneum.repair.R
+import com.iteneum.repair.data.RepairViewModel
 
 /**
  * Function that creates [RepairView] compose. for user's service request.
  * User will require to fill all the correspondent fields on the screen.
  *
+ * Function [RepairView] has
+ * @param navigateToApartment - to get the function to navigate ApartmentView when clicking button
+ * @param repairViewModel - to get the Repair View Model access
+ *
+ * Function [RepairContainer] has
+ * @param dataInfo - to use if for displaying Unit in the Unit Text Field
+ * @param repairViewModel -  to get the Repair View Model access
+ * @param navigateToApartment - its passed from RepairView() for same purpose
+ *
  * @author Jose Miguel Garcia Reyes
  */
 
+@Composable
+fun RepairView(
+    navigateToApartment: () -> Unit,
+    repairViewModel: RepairViewModel = hiltViewModel()
+) {
+    LaunchedEffect(true) {
+        repairViewModel.getInformation()
+    }
+    RepairContainer(
+        dataInfo = repairViewModel.dataInfo.value,
+        navigateToApartment = navigateToApartment
+    )
+}
+
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
-fun RepairView() {
+fun RepairContainer(
+    dataInfo: RepairModel?,
```

Comment:
You are not using this variable: `dataInfo` to show the information when the view charged

Reply:
Yes, you are right. But I wrote on the TODO:
/* TODO - LpOutlinedTextFieldInput (Check component required) - To enable to action of passing value argument --> value = dataInfo?.unitDepartment ?: "000" */

There is a problem related to obtain the Unit (“Department number”) from the data source and paste it on the Repair View (as desired by Sergio). The component used to have the Unit does not have the actual “value” parameter. We need to add it so we can use it. This involves however the modification of the actual component.

The parameter value is not on the function parameters. It is required to be able to pass and use the dataInfo.
![image](https://user-images.githubusercontent.com/25538206/236571781-a0ebce86-27a5-4f6b-a4e2-8008d7a5964a.png)